lib/philomena_web/controllers/image/tag_change_controller.ex

```diff
@@ -47,34 +44,16 @@ defmodule PhilomenaWeb.Image.TagChangeController do
       details: &log_details/2,
       data: %{image: image, details: tag_change_details(tag_change)}
     )
-    |> redirect(to: ~p"/images/#{image}")
+    |> redirect(to: ~p"/images/#{image}/tag_changes")
   end
 
-  defp added_filter(query, %{"added" => "1"}),
-    do: where(query, added: true)
-
-  defp added_filter(query, %{"added" => "0"}),
-    do: where(query, added: false)
-
-  defp added_filter(query, _params),
-    do: query
-
   defp log_details(_action, %{image: image, details: details}) do
     %{
-      body: "Deleted tag change #{details} on image #{image.id} from history",
-      subject_path: ~p"/images/#{image}"
+      body: "Deleted tag change batch #{details} on image #{image.id} from history",
+      subject_path: ~p"/images/#{image}/tag_changes"
     }
   end
 
-  defp tag_change_details(%TagChange{added: true, tag: %Tag{name: tag_name}}),
-    do: "+#{tag_name}"
-
-  defp tag_change_details(%TagChange{added: true, tag_name_cache: tag_name}),
-    do: "+#{tag_name}"
-
-  defp tag_change_details(%TagChange{added: false, tag: %Tag{name: tag_name}}),
-    do: "-#{tag_name}"
-
-  defp tag_change_details(%TagChange{added: false, tag_name_cache: tag_name}),
-    do: "-#{tag_name}"
+  defp tag_change_details(%{user: %{name: name}, tags: tags}),
+    do: "by #{name} containing #{Enum.count(tags)} change(s)"
```

Comment:
Shouldn't say "batch" for consistency (this function is also no longer necessary)
```"Deleted tag change by #{name} containing #{Enum.count(tags)} tags on image ... from history```